### DISCUSSION TITLE
To create a new repository from the GitHub application

### DISCUSSION THREAD

**Discussion by RichKame** [community/community#144498] [Mobile] [answered]
How do I create a new repository from the iOS GitHub app?

When I try to create a new one from the + button, I can only add new issues

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, バージョン 1.185.0 (146370144)
Device: iPhone17,2 (iOS 18.1)
</details>

**Comment by HenSquared** [comment, marked as answer]
Hi. Creating repositories from the Mobile app is not supported right now. I've created an issue to track this request. 

Thanks!

**Reply by AnarchyX67** [reply]
It's 2026, and many people work from their phones. How is it not an option in the iOS app to start a new repository?

**Reply by HenSquared** [reply]
We support this now:
https://github.blog/changelog/2026-05-11-create-repositories-on-the-go-with-github-mobile/
